packages/syft/src/syft/serde/recursive.py

```diff
@@ -269,11 +269,11 @@ def rs_proto2object(proto: _DynamicStructBuilder) -> Any:
     # clean this mess, Tudor
     module_parts = proto.fullyQualifiedName.split(".")
     klass = module_parts.pop()
-    class_type: Type = type(None)
+    class_type: Type[Any] = type(None)
```

Comment:
Is this needed? Isn't `Type` is the same as `Type[Any]`?

Reply:
I think it's mostly the same but `Type[Any]` is more detailed and explicitly states that `class_type` can hold any class

Reply:
For generic type usually `Any` is omitted. For example we usually just write `List`, `Dict` instead of `List[Any]` and `Dict[Any, Any]`. If `Type[Any]` is the same as `Type` we should just use the latter for consistency, unless there is a difference that I'm not aware of.

Reply:
Yeah sure sounds good. Thanks @kiendang